src/object/thunderstorm.hpp

```diff
@@ -60,7 +61,7 @@ class Thunderstorm final : public GameObject,
   void thunder();
 
   /** Play lightning, i.e. call flash() and electrify() */
-  void lightning();
+  void lightning(bool is_scripted = false);
```

Comment:
When we migrate to `simplesquirrel`, the `bool is_scripted` parameter wouldn't be supported - the scripting function to be exposed should rather be a separate function in the code, preferably with the same name as the one it's exposed with (easier for documentation).